examples/next/docs/vue/hot-column-component/src/assets/styles.css

```diff
@@ -0,0 +1,11 @@
+body {
+    padding: 1rem;
+    margin: 0;
+}
+
+#app {
+    overflow: auto;
+    min-width: 100%;
+    min-height: 100%;
+    height: calc(100vh - 2rem);
+}
```

Comment:
Note to self: test the CSS, looks like `body` should be enough. But many examples has this additional CSS. Use case scenarios: iframe, full screen. Check how does it work with 1000x1000 columns and rows, if Handsontable does not need additional `width` and `height` settings. ie "auto"